### PR TITLE
add docs for v5.1: reference -> CLI 

### PR DESCRIPTION
**Pull request by marsishandsome**
### What is changed?

TiKV's official website is planned to be refactored as discussed in https://github.com/tikv/website/issues/227.

This PR covers `Reference` -> `CLI`

This PR references to
- https://docs.pingcap.com/tidb/stable/pd-control
- https://docs.pingcap.com/tidb/stable/tikv-control
- https://docs.pingcap.com/tidb/stable/pd-recover

### Any related PRs or issues?

https://github.com/tikv/website/issues/227.

### Which version does your change affect?
v5.1

### CODE REVIEW

**Comment by marsishandsome** [comment]
@andylokandy @zz-jason @iosmanthus PTAL

File: content/docs/5.1/reference/CLI/pd-recover.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+`{version}` indicates the version number of TiKV. For example, if `{version}` is `v5.0.0`, the package download link is `https://download.pingcap.org/tidb-v5.0.0-linux-amd64.tar.gz`.
+{{< /info >}}
+
+## Quick Start
+
+This section describes how to use PD Recover to recover a PD cluster.
+
+### Get cluster ID
+
+The cluster ID can be obtained from the log of PD, TiKV. To get the cluster ID, you can view the log directly on the server.
```
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md:39]: sure

File: content/docs/5.1/reference/CLI/pd-recover.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+```bash
+[2019/10/14 07:06:35.278 +00:00] [INFO] [tikv-server.rs:464] ["connect to PD cluster 6747551640615446306"]
+...
+```
+
+### Get allocated ID
+
+The allocated ID value you specify must be larger than the currently largest allocated ID value. To get allocated ID, you can either get it from the monitor, or view the log directly on the server.
```
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md:69]: sure

File: content/docs/5.1/reference/CLI/pd-recover.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+---
+title: pd-recover
+description: Use PD Recover to recover a PD cluster which cannot start or provide services normally.
+menu:
+    "5.1":
+        parent: CLI
+        weight: 3
+---
+
+PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally.
```
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md:10]: good suggestion

[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md]: fixed, PTAL

File: content/docs/5.1/reference/CLI/pd-recover.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md:78]: fixed, PTAL

File: content/docs/5.1/reference/CLI/pd-recover.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/reference/CLI/pd-recover.md:48]: fixed, PTAL

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**